src/DssSpell.sol

```diff
@@ -63,6 +67,11 @@ contract DssSpellAction is DssAction {
     uint256 public constant ARB_MAX_SUBMISSION_COST = 1e14;
     uint256 public constant ARB_L1_CALL_VALUE = ARB_MAX_SUBMISSION_COST + ARB_MAX_GAS * ARB_GAS_PRICE_BID;
 
+    // see: https://github.com/makerdao/starknet-spells-goerli/tree/teleport-spell#estimate-l1-l2-fee
+    uint256 public constant STA_FEE_ESTIMATION = 104825981483660;
```

Comment:
Some questions:

1. If this value is not enough would it fail on l1 or just not be included on l2?

2. In case it is just not included in l2, can it be retried permissionlessly somehow with a larger gas fee? (in Optimism and Arbitrum it can in case it reverts on l2 for not enough gas).

3. It seems that we just use the estimated result, while in practice the spell will be executed more than 2 days later. For Abitrum there is a 20x factor for that now, do we need one here as well?

Reply:
1. If there is not enough gas spell will 'fail' on l2, from l1 perspective l2 part of the spell will seem to not have been executed. 
2. Right now there is no permissionless mechanism to retry failed l1->l2 message. It would need to be built into the L1 gov relay using message cancelation. As there is a large upgrade coming in next few months to the way Starknet handles failed transactions we felt it was not worth the complication.
3. We need a safety buffer here as well. That is why we have STA_FEE_BUFFER. How the 20x was deceided?

Reply:
The problem with setting excess fee for l1->l2 calls in Starknet to acomodate gas price changes is that currently unused fees are not returned.